demo/index.html

```diff
@@ -1,61 +1,74 @@
-<!doctype html>
 <!-- Copyright Eric Bidelman <ebidel@gmail.com> -->
+
+<!doctype html>
 <html>
-<head>
-  <title>geo-location Demo</title>
-  <script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
-  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300|Source+Code+Pro">
-  <link rel="import" href="../geo-location.html">
-  <link rel="import" href="../../iron-flex-layout/classes/iron-flex-layout.html">
-  <style>
-  * {
-    box-sizing: border-box;
-  }
-  html, body {
-    height: 100%;
-    padding: 0;
-  }
-  body {
-    font-family: 'Roboto', sans-serif;
-    background: black;
-    color: white;
-  }
-  h2 {
-    font-weight: 300;
-    font-size: 3.5vw;
-    margin-top: 0;
-  }
-  pre {
-    font-family: 'Source Code Pro';
-  }
-  </style>
-</head>
-<body class="layout vertical center-center" unresolved>
-
-
-<template is="dom-bind">
-  <geo-location watch-pos latitude="{{latitude}}" longitude="{{longitude}}"></geo-location>
-  <h2><pre>&lt;geo-location latitude="<span>{{latitude}}</span>"
-              longitude="<span>{{longitude}}</span>">
-&lt;/geo-location></pre></h2>
-</template>
-
-<div>Position updates as your device moves</div>
-
-<script>
-var t = document.querySelector('template');
-t.latitude = '???';
-t.longitude = '???';
-
-</script>
-<script>
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
- (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
- m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
- })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
- ga('create', 'UA-43475701-1', 'ebidel.github.io');
- ga('send', 'pageview');
-</script>
-</body>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>geo-location demo</title>
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../../google-map/google-map.html">
+    <link rel="import" href="../geo-location.html">
+
+    <custom-style>
+      <style is="custom-style" include="demo-pages-shared-styles">
+        google-map {
+          height: 600px;
+        }
+      </style>
+    </custom-style>
+  </head>
+  <body>
+    <div class="vertical-section-container centered">
+      <h3>Get the device geolocation values</h3>
+      <demo-snippet>
+        <template>
+          <dom-bind>
+            <template is="dom-bind">
+              <geo-location
```

Comment:
@ebidel Done, but I believe it's not a good idea, because it's less readable. IMHO, if an element (maybe except repeater elements like `<dom-repeat>`) has more than one attribute, the attributes should be on separate lines. Elements indents with 2 spaces, but properties with 4 spaces.

@ebidel @abdonrd Is there any _official_ code style guideline for web components?